### DISCUSSION TITLE
Pin comments inside discussions

### DISCUSSION THREAD

**Discussion by TrueBrain** [community/community#3003] [Discussions]
While experimenting with the new Discussions feature, we quickly ran into the issue that some comments on discussions are more useful than others. I would like to be able to put focus on certain comments.

Some solutions I can think of:
- Allow to pin one or more comments
- Allow changing order of comments (as they all have their own replies, the order of comments doesn't always matter .. if someone was replying to another comment, they should have used reply, not a new comment :D).

But possibly a UX'r can come up with a much better idea :D

**Comment by nikolqy** [comment]
Absolutely. I was just thinking that same thing!

**Comment by Fatman13** [comment]
Anyone working on this feature yet?
